lib/core/src/Data/Quantity.hs

```diff
@@ -154,6 +161,9 @@ instance FromText Percentage where
         err = TextDecodingError
             "expected a value between 0 and 100 with a '%' suffix (e.g. '14%')"
 
+instance Buildable Percentage where
+    build = Builder.fromText . toText
```

Comment:
```suggestion
build = build . toText
```

Reply:
Nope, I would have tried that:
```
    • Couldn't match expected type ‘Percentage -> Builder.Builder’
                  with actual type ‘Builder.Builder’

```

- `build :: Builder a -> a`
- `toText :: ToText a => a -> Text`

Reply:
Hmmm. No, 

```
class Buildable p where 
    build :: p -> Builder
```

and we have `Buildable Text`, so we can `build (_ :: Text) :: Builder`

Reply:
🤔 🙃 👍